src/components/PetSidebar.jsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import {
+  SideBarSection,
+  SummaryList
+} from "@baltimorecounty/dotgov-components";
+
+const publicWorksSummaryItems = [
```

Comment:
I think this matches up with XD, this should be animal services stuff. I will shoot a message to you and sam so we can get this content to be more realistic.

Reply:
I believe tim is working with sam

Reply:
this is updated per discussion with Sam. Noticed that email address does not wrap inside of component sidebar so this is a bug with the component.